galaxy_importer/ansible_test/runners/local.py

```diff
@@ -16,10 +16,48 @@
 # along with Galaxy.  If not, see <http://www.apache.org/licenses/>.
 
 
+import os
+import subprocess
+
 from galaxy_importer.ansible_test.runners.base import BaseTestRunner
 
 
 class LocalAnsibleTestRunner(BaseTestRunner):
     """Run ansible-test locally with --docker or using venv."""
     def run(self):
-        pass
+        version_proc = subprocess.Popen(
+            ['ansible', '--version'],
+            stdout=subprocess.PIPE,
+            encoding='utf-8',
+        )
+        self.log.info(f'Using {list(version_proc.stdout)[0].rstrip()}')
+
+        suffix = f'ansible_collections/{self.metadata.namespace}/{self.metadata.name}/'
+        collection_dir = os.path.join(self.dir, suffix)
+
+        cmd = [
+            'ansible-test', 'sanity',
+            '--docker',
+            '--color', 'yes',
+            '--failure-ok',
+        ]
+
+        self.log.info(
+            'Running ansible-test sanity on '
+            f'{self.metadata.namespace}-{self.metadata.name}-{self.metadata.version} ...')
+        self.log.info(f'{" ".join(cmd)}')
+
+        proc = subprocess.Popen(
+            cmd,
+            cwd=collection_dir,
+            stdout=subprocess.PIPE,
+            stderr=subprocess.STDOUT,
+            encoding='utf-8',
+        )
+
+        for line in proc.stdout:
+            self.log.info(line.strip())
+
+        if proc.wait() != 0:
+            msg = 'An exception occurred in ansible-test sanity'
```

Comment:
Use the actual cmd line that was run here instead of just 'ansible-test sanity'

ie some variant of 

``` python
self.log.error('An exception occurred in %s', ' '.join(cmd))
# or
self.log.error('An exception occurred in {}', ' '.join(cmd))
# or if py3 only
self.log.error('An exception occurred in {}', proc.args)
```

Reply:
return code and/or stderr wouldn't hurt either. Though ansible-test stderr can be kind of knarly iirc, so maybe not in that case.

Reply:
Oh, and possibly the collection_dir or info indicating which collection failed. Whatever info you need to debug why/what failed when someone cuts and pastes the error into an issue.